_inc/lib/core-api/wpcom-endpoints/gutenberg-available-extensions.php

```diff
@@ -0,0 +1,61 @@
+<?php
+
+/*
+ * Plugin Name: Available Gutenberg Extensions (Blocks and Plugins) for wpcom/v2 WP-API
```

Comment:
Is this a plugin? I didn't expect to see a plugin docblock here.

Reply:
I quit as a reviewer

Reply:
Copypasta strikes again :roll_eyes: